### ISSUE TITLE
Packaging Request: scli

### ISSUE THREAD

**Issue by herrmoltke** [NixOS/nixpkgs#120615]
**Project description**
scli is a simple terminal user interface for Signal.

**Metadata**

* homepage URL: https://github.com/isamert/scli
* source URL: https://github.com/isamert/scli
* license: gpl3
* platforms: linux

**Comment by alex-eyre** [comment]
So I'm looking for some more experienced nixpkgs people here, because scli relies on the fact you have signal-cli in your PATH at runtime to do login management etc. but I can't figure out how to ensure it's installed alongside scli, i.e. have signal-cli be in `result/bin` alongside scli, without then conflicting with the signal-cli package itself.
My first attempt is here: #122315

**Comment by Stunkymonkey** [comment]
i guess this can be closed because of #122315

**Comment by herrmoltke** [comment]
I am not sure where the issue is, but `scli` immediately segfaults whenever I run it with any combination of subcommands:

```
> scli
fish: “scli” terminated by signal SIGSEGV (Address boundary error)
```

The same thing also happens with `scli`'s dependency, `signal-cli`.

NB: I am using the hardened Linux kernel.
